packages/app/components/Queue/Student/QuestionForm.tsx

```diff
@@ -136,6 +137,14 @@ export default function QuestionForm({
     }
   };
 
+  useHotkeys("enter", () => onClickSubmit(), { enableOnTags: ["TEXTAREA"] }, [
```

Comment:
`enableOnTags: ["TEXTAREA"]` allows the `enter` hotkey to be used whether or not the text form is focused